src/components/HomePage/SearchArea/SearchBar.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import IconButton from '@material-ui/core/IconButton';
+import Search from '@material-ui/icons/Search';
+
+function SearchBar(props) {
+
+    const {searchValue, updateSearchValue, className, submitSearchForm} = props;
+    
+    const handleChange = e => {
+        updateSearchValue(e.target.value);
+    };
+
+    const handleButtonClick = e => {
+        e.preventDefault();
+        submitSearchForm(e);
+    };
+
+    return (
+        <TextField
+            label="Search"
+            name="search"
+            margin="dense"
+            value={searchValue}
+            onChange={handleChange}
+            className={className}
+            InputProps={{
+                "endAdornment": 
+    <InputAdornment position="end">
```

Comment:
Uh oh, eslint spaghetti?

Reply:
Looks like weird indentation now, because of being nested and a value of a key... Is there a better way to pass that prop? This looks ugly

Reply:
Definitely eslint spaghett, at the rules definition time, we could not find a reliable way of effectively manage the components props, maybe the situation has changed now. Needs to be reviewed.

Reply:
Tracked here: https://github.com/NIAEFEUP/nijobs-fe/issues/16